app/javascript/react/screens/App/Overview/OverviewConstants.js

```diff
@@ -19,8 +19,7 @@ export const ARCHIVE_TRANSFORMATION_PLAN = 'ARCHIVE_TRANSFORMATION_PLAN';
 export const DELETE_TRANSFORMATION_PLAN = 'DELETE_TRANSFORMATION_PLAN';
 
 export const FETCH_PROVIDERS = 'FETCH_PROVIDERS';
-export const FETCH_PROVIDERS_URL =
-  '/api/providers?expand=resources&attributes=type&filter[]=or%20type=ManageIQ::Providers::Redhat::InfraManager&filter[]=type=ManageIQ::Providers::Vmware::InfraManager';
+export const FETCH_PROVIDERS_URL = '/api/providers?expand=resources';
```

Comment:
Using a filter will give a leaner payload, while `'/api/providers?expand=resources'` will keep it generic.

Both have their pros and cons hence I am OK with either one.